OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -1785,9 +1793,14 @@ public enum CssValueType
 
     public class CssParser
     {
-        private List<string> m_errors = new List<string>();
+        public IErrorHandler ErrorHandler { get; private set; }
```

Comment:
you can remove the `private set` for a readonly property

Reply:
done